projects/angular-calendar/src/modules/month/calendar-month-row.component.ts

```diff
@@ -150,39 +146,37 @@ export class CalendarMonthRowComponent implements OnChanges {
   manageWidth(note: CalendarEvent): string {
     let width: number;
     const indexStartDate = this.daysSliced
-      .map((daySliced) =>
-        this.timezoneManager.reverseTz(daySliced.date, this.timezone)
-      )
-      .findIndex(
-        (date) =>
-          new Date(date.toDateString()).valueOf() === note.start.valueOf()
-      );
+      .map((daySliced) => daySliced.date)
+      .findIndex((date) => date.toDateString() === note.start.toDateString());
     const indexEndDate = this.daysSliced
-      .map((daySliced) =>
-        this.timezoneManager.reverseTz(daySliced.date, this.timezone)
-      )
-      .findIndex(
-        (date) => new Date(date.toDateString()).valueOf() === note.end.valueOf()
-      );
+      .map((daySliced) => daySliced.date)
+      .findIndex((date) => date.toDateString() === note.end.toDateString());
+    console.log(
```

Comment:
:)

Reply:
i've removed logs + removed the use of the timezone manager on other places (cause i captured some errors on some timezones) so have another look ;)

Reply:
Sounds good to me, notes start and end are points in time and the boundaries of the day/week based on the timezone are already calculated somewhere else.
We shouldn't have to bother on timezone at this level.